dotnet-local.cmd

```diff
@@ -0,0 +1,9 @@
+@echo off
+SET ROOT=%~dp0
+IF EXIST "%ROOT%\bin\dotnet\dotnet.exe" (
+    SET DOTNET_ROOT=%ROOT%\bin\dotnet
+    SET PATH=%DOTNET_ROOT%;%PATH%
+    call "%ROOT%\bin\dotnet\dotnet.exe" %*
+) ELSE (
+    echo "You need to run 'build.ps1' first."
```

Comment:
/cc @PureWeen I'm not sure we use those `build.ps1/sh` scripts or if they are vestigial? I think we could just say, "you need to build MAUI first" and link to:

https://github.com/dotnet/maui/blob/main/.github/DEVELOPMENT.md

Reply:
On the one hand, sure, but on the other hand… I find that `.github/DEVELOPMENT.md` page to be inscrutable.  *How* should I build dotnet/maui if I *don't* use `build.sh`/`build.ps1`?

And if `build.sh`/`build.ps1` *shouldn't* be used, shouldn't we just *delete them*?

Reply:
Yes, I don't know why they exist if `DEVELOPMENT.md` doesn't mention them?

Reply:
We were using `build.sh` and `build.ps1` before dotnet had a cake tool and before anyone that didn't work here could even hope to build MAUI. 

We could probably just delete them at this point